client/src/App.js

```diff
@@ -1,11 +1,49 @@
 import React, { Component } from 'react';
 import SideNav from './components/SideNav';
+import styled from 'styled-components';
+import { Route } from 'react-router-dom';
+import ListView from './components/ListView';
+
+const MainContent = styled.div`
+  padding: 4rem;
+`;
+
+const textBody = `Lorem ipsum dolor sit amet, at essent consequat mediocritatem nam, sea an rebum graecis. His autem simul percipitur ne. Quod eirmod suavitate ne vix, sed eu detracto antiopam. Ad putent quaerendum scriptorem eos, est id principes consetetur. Ornatus nominavi vis te, at vel possit oportere argumentum. Semper evertitur pri id. Dolorum complectitur ad vim, at sea pertinax moderatius inciderint, reque etiam mel ei. In mea congue eripuit phaedrum, vix eros ignota eirmod ne. Vel lorem putent reprehendunt te. Usu ad debitis antiopam, mel aeque moderatius eu.
```

Comment:
I noticed another student did this too. :) That's a good way of not repeating too much content manually.